README.rst

```diff
@@ -393,6 +393,19 @@ If you want to run Nitpick as a flake8 plugin instead::
           - id: flake8
             additional_dependencies: [nitpick]
 
+Run as a MegaLinter plugin
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
The extra tilde doesn’t matter, not to the restructuredText parser. :-)

They must be tildes however otherwise this becomes a different level header! The intent is for this header to be the same level as the pre-commit section before it. Take a look at the [rendered version](https://github.com/andreoliwa/nitpick/blob/dc3a2762641ed4839e5d0fa09003ef4df7ab5258/README.rst#run-as-a-megalinter-plugin).